ballerina/Package.md

```diff
@@ -1,18 +1,63 @@
+The Ballerina `jwt` package facilitates auth providers that are to be used by the clients and listeners of different protocol connectors. Also, it provides the APIs for issuing a self-signed JWT and validating a JWT.
 
-The Ballerina `jwt` module facilitates auth providers that are to be used by the clients and listeners of different protocol connectors. Also, it provides the APIs for issuing a self-signed JWT and validating a JWT.
+### Listener JWT Auth Provider
 
-### Report Issues
+Represents the listener JWT Auth provider, which is used to authenticate the provided credentials (JWT) against the provided JWT validator configurations.
 
-To report bugs, request new features, start new discussions, view project boards, etc., go to the <a target="_blank" href="https://github.com/ballerina-platform/ballerina-standard-library">Ballerina standard library parent repository</a>.
+### Client JWT Auth Provider
 
-### Useful Links
+Represents the client JWT Auth provider, which is used to authenticate with an external endpoint by issuing a self-signed JWT against the provided JWT issuer configurations.
 
-- Chat live with us via our <a target="_blank" href="https://ballerina.io/community/slack/">Slack channel</a>.
-- Post all technical questions on Stack Overflow with the <a target="_blank" href="https://stackoverflow.com/questions/tagged/ballerina">#ballerina</a> tag.
+### JWT Issuer
+
+A self-signed JWT can be issued with the provided configurations using this API as follows:
+
+```ballerina
+jwt:IssuerConfig issuerConfig = {
+    username: "ballerina",
+    issuer: "wso2",
+    audience: "vEwzbcasJVQm1jVYHUHCjhxZ4tYa",
+    expTime: 3600,
+    signatureConfig: {
+        config: {
+            keyFile: "resource/path/to/private.key"
+        }
+    }
+};
+
+string jwt = check jwt:issue(issuerConfig);
+```
+
+### JWT Validator
+
+A JWT can be validated with the provided configurations using the API as follows:
+
+```ballerina
+string jwt = "eyJ0eXAiOiJKV1QiLA0KI[...omitted for brevity...]mB92K27uhbwW1gFWFOEjXk";
+
+jwt:ValidatorConfig validatorConfig = {
+    issuer: "wso2",
+    audience: "vEwzbcasJVQm1jVYHUHCjhxZ4tYa",
+    clockSkew: 60,
+    signatureConfig: {
+        certFile: "../resource/path/to/public.crt"
+    }
+};
+
+jwt:Payload result = check jwt:validate(jwt, validatorConfig);
+```
+
+## Report Issues
+
+To report bugs, request new features, start new discussions, view project boards, etc., go to the [Ballerina standard library parent repository](https://github.com/ballerina-platform/ballerina-standard-library).
+
+## Useful Links
```

Comment:
Don't we need the link to docs and BBE

Reply:
We only have these in the `README.md`, but not in the `Package.md`. Right? If it is needed, I will add those to `Package.md` as well.

Reply:
Yeah, we don't need it. We can add those later as central evolves.

Reply:
Ack.